Sources/API/Debug.swift

```diff
@@ -16,58 +16,122 @@ public class Debug {
     private init() {}
 }
 
-private func withPrint(f: () -> String) -> String {
-    let string = f()
-    Swift.print(string)
-    return string
-}
-
 // MARK: Standard API
 
 extension Debug {
     /// Output `target` to console.
     /// - Parameters:
     ///   - target: target
-    /// - Returns: String that is the same as output console.
-    @discardableResult
-    public static func print(_ target: Any) -> String {
-        withPrint {
-            Pretty(formatter: SinglelineFormatter()).string(target, debug: false)
-        }
+    public static func print(
```

Comment:
Abolish return `String`, because prevent display in two times in LLVM commands.

For example in before this PR:
![image](https://user-images.githubusercontent.com/2990285/76469850-7b6b6680-6432-11ea-8d63-6871e1639a0a.png)

This behavior is not prefererd. Therefore I decided, remove return `String` :)